perk-store-services/src/main/java/org/exoplatform/perkstore/dao/PerkStoreOrderDAO.java

```diff
@@ -335,5 +335,9 @@ private long getEndOfDayMillis(long selectedDate) {
     long startTimeOfDay = date.atStartOfDay().toEpochSecond(ZoneOffset.UTC);
     return startTimeOfDay * 1000;
   }
-
+  public List<ProductOrderEntity> getOrdersByIdentity(long identityId) {
```

Comment:
Why don't we use getOrders directly and buuld the query there ? isn't it possible this way ?